### PR TITLE
EventFiltering/PWGEM: add a trigger decision for pcm w wire

### PR DESCRIPTION
**Pull request by dsekihat**
EventFiltering/PWGEM: add a trigger decision for pcm w wire

### CODE REVIEW

File: EventFiltering/PWGEM/EMPhotonFilter.cxx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "Common/DataModel/CaloClusters.h"
+#include "DataFormatsPHOS/TriggerRecord.h"
+#include "PWGEM/PhotonMeson/DataModel/gammaTables.h"
+#include "PWGEM/PhotonMeson/Core/V0PhotonCut.h"
+#include "PWGEM/PhotonMeson/Core/CutsLibrary.h"
+#include "../filterTables.h"
```
Comment (mhemmer-cern) [inline EventFiltering/PWGEM/EMPhotonFilter.cxx:22]: Please don't use relative paths with `..`

File: EventFiltering/PWGEM/EMPhotonFilterQC.cxx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "Common/DataModel/CaloClusters.h"
+#include "DataFormatsPHOS/TriggerRecord.h"
+#include "PWGEM/PhotonMeson/DataModel/gammaTables.h"
+#include "PWGEM/PhotonMeson/Core/V0PhotonCut.h"
+#include "PWGEM/PhotonMeson/Core/CutsLibrary.h"
+#include "../filterTables.h"
```
Comment (mhemmer-cern) [inline EventFiltering/PWGEM/EMPhotonFilterQC.cxx:22]: Same here

**Review by mhemmer-cern — DISMISSED** [review]

File: EventFiltering/PWGEM/EMPhotonFilterQC.cxx (hunk 2 above)
Comment (dsekihat) [reply EventFiltering/PWGEM/EMPhotonFilterQC.cxx]: Thank you. It is fixed.

**Review by mhemmer-cern — APPROVED** [review]

**Comment by dsekihat** [comment]
Dear @mpuccio , @strogolo , could you have look at this PR, please?

**Pull request closed without merging**